.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,80 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
+        with:
+          script: |
+            // Update the following two constants each year
+            const FILTER_LABEL = '2020 chapter'
+            const TRACKER_ISSUE_ID = 989
```

Re-use the env variables set above.

```suggestion
            const FILTER_LABEL = process.env.FILTER_LABEL
            const TRACKER_ISSUE_ID = process.env.TRACKER_ISSUE_ID
```

---

Done!